samples/keys/hw_unique_key/README.rst

```diff
@@ -9,41 +9,38 @@ Hardware unique key
 
 This sample shows how to use a hardware unique key (HUK) to derive an encryption key through psa_crypto APIs.
 
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf5340dk_nrf5340_cpuapp, nrf5340dk_nrf5340_cpuapp_ns, nrf9160dk_nrf9160, nrf9160dk_nrf9160_ns, nrf52840dk_nrf52840
+
 Overview
 ********
 
 The sample goes through the following steps to derive a key and use it to encrypt a string:
 
-1. Initializing the hardware, if applicable.
+1. Initializes the hardware, if applicable.
 
    When TF-M is enabled, the hardware is initialized automatically when booting.
 
-#. Generating HUKs, if applicable.
+#. Generates HUKs, if applicable.
 
-   The sample generates a random HUK and writes it to KMU or flash. When no
-   KMU is present, the device reboots to allow the immutable bootloader to feed
-   the key into CryptoCell.
-   When TF-M is enabled, the HUKs are generated automatically when booting TF-M,
-   and no action is needed in the sample.
+   The sample generates a random HUK and writes it to the Key Management Unit (KMU) or flash.
+   When no KMU is present, the device reboots to allow the immutable bootloader to feed the key into CryptoCell.
+   When TF-M is enabled, the HUKs are generated automatically when booting TF-M, and no action is needed in the sample.
 
-#. Generating a random initialization vector (IV).
+#. Generates a random initialization vector (IV).
 
-#. Deriving a key used for encryption.
+#. Derives a key used for encryption.
 
    When TF-M is enabled, the key is derived using the psa_crypto APIs.
-   Otherwise, the native nrf_cc3xx_platform APIs are used, and the key is
-   imported into psa_crypto.
-
-#. Encrypting a sample string using the key_id received from the psa_crypto.
+   Otherwise, the native nrf_cc3xx_platform APIs are used, and the key is imported into psa_crypto.
 
-Requirements
-************
-
-The sample supports the following development kits:
-
-.. table-from-rows:: /includes/sample_board_rows.txt
-   :header: heading
-   :rows: nrf5340dk_nrf5340_cpuapp, nrf5340dk_nrf5340_cpuapp_ns, nrf9160dk_nrf9160, nrf9160dk_nrf9160_ns, nrf52840dk_nrf52840
+#. Encrypting a sample string using the ``key_id`` received from the psa_crypto.
```

Comment:
Should this be 'Encrypts' so it matches the previous bullet points?

Reply:
Absolutely yes. I was trying to be too fast. Thanks for spotting this. Fixed now.